### DISCUSSION TITLE
Will not run after install

### DISCUSSION THREAD

**Discussion by eb99** [tomshaffner/PiThermalCam#16] [Q&A] [answered]
Hi,
I am abandoning my previous topic as this is off topic now.

I was trying to toubleshoot autostart problems, and went installing packages it thought were missing, but this ended up breaking it.
so I wiped it and started again with fresh OS install, did the apt-get update and upgrade, then followed the install instructions on your page exactly, and it will no longer run.
I suspect that some of the dependant packages have been updated recently and this is interfering with the program.
Can you please have a look and see if this is the case?
I don't have the knowledge or experience with python to debug it.
Thanks

**Comment by tomshaffner** [comment, marked as answer]
@eb99 , sorry for the long delay. Actually I just tried it again with a fresh install of the newest PI OS, and it turns out the normal install instructions actually have an issue with numpy being too _old_. In order to get it running I had to update numpy manually beyond the default using `pip3 install -U numpy`. That was the only additional instruction needed (beyond the pre-requisites and other instructions in the page) to get it up and running. I'd suggest trying that and giving it another go; I'll be it works for you then.

And if no, please post pictures or excerpts of the error messages you get.
